contrib/completion/zsh/_docker

```diff
@@ -2527,6 +2527,8 @@ __docker_volume_subcommand() {
         (prune)
             _arguments $(__docker_arguments) \
                 $opts_help \
+                "($help -a --all)"{-a,--all}"[Remove all unused volumes, not just anonymous ones]" \
```

Comment:
nit: perhaps it's a bit redundant, but just to be consistent with the warning messages which indicates the term `local`:

```suggestion
                "($help -a --all)"{-a,--all}"[Remove all unused local volumes, not just anonymous ones]" \
```

Reply:
done 👍